.changeset/loud-clouds-guess.md

```diff
@@ -0,0 +1,6 @@
+---
+'@leafygreen-ui/tooltip': patch
+---
+
+- `Tooltip` typography previously relied on consumer application's styles. Now, Tooltip children are wrapped in LeafyGreen Body component. 
+- Setting `usePortal` to `false` no longer stops Tooltips from appearing on hover when the trigger is a LeafyGreen Button component.
```

Comment:
Will adjust the changeset once there's alignment on the approach of updating Button rather than baking this change into Tooltip directly.